blockly-ws-search/README.md

```diff
@@ -0,0 +1,32 @@
+[Home](../README.md)
+
+# blockly-ws-search [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```

Comment:
Use ``@blockly/plugin-workspace-search`` here to ensure consistency when referring to this package.